src/java.desktop/macosx/native/libjsound/PLATFORM_API_MacOSX_Ports.cpp

```diff
@@ -635,7 +635,7 @@ void PORT_GetControls(void* id, INT32 portIndex, PortControlCreator* creator) {
                     if (channelName == NULL) {
                         return;
                     }
-                    sprintf(channelName, "Ch %d", ch);
+                    os::snprintf(channelName, 16, "Ch %d", ch);
```

Comment:
You can't use this here - this is not hotspot code!

Reply:
You are right.  Reverted to use `snprintf` for desktop update.  Thanks!